ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageCve/ImageCvePage.tsx

```diff
@@ -382,9 +382,7 @@ function ImageCvePage() {
             />
             <PageSection variant="light" className="pf-v5-u-py-md">
                 <Breadcrumb>
-                    <BreadcrumbItemLink to={workloadCveOverviewCvePath}>
-                        {pageTitle}
-                    </BreadcrumbItemLink>
+                    <BreadcrumbItemLink to={workloadCveOverviewCvePath}>CVEs</BreadcrumbItemLink>
```

Comment:
Observation, not change request. The pages for the breadcrumb link:
* User workload vulnerabilities
* Platform violations

Expecially on my mind because planning for CVE/advisory separation and GHSA will not be separated.